_sass/layouts/_search.scss

```diff
@@ -5,9 +5,9 @@
 }
 
 .layout-search {
-  h1 {
-    font-size: 24px;
-  }
+  // h1 {
+  //   font-size: 24px;
+  // }
```

Comment:
A general preference of mine is to avoid leaving in commented code, unless you're sure you want to switch back at some point in the future.

Reply:
Very good point. I'm removing them.